Chat/TokenManagement/UserTokenManager.cs

```diff
@@ -28,5 +29,26 @@ public async Task<(string userMri, string skypeToken, long expiresDateUtc)> Gene
                 return (null, null, -1);
             }
         }
+
+        public async Task<(string userMri, string skypeToken, long expiresDateUtc)> RefreshTokenAsync(string resourceConnectionString, string identity)
+        {
+            try
+            {
+                CommunicationIdentityClient communicationIdentityClient = new CommunicationIdentityClient(resourceConnectionString);
```

Comment:
nit: C# with var looks nicer imho
```suggestion
                var communicationIdentityClient = new CommunicationIdentityClient(resourceConnectionString);
```

Reply:
making the change and now I can simplify the imports